### PR TITLE
Set the `minServerVersion` requirement to 4.9 in isMaster-network/command-error tests

### PR DESCRIPTION
**Pull request by stIncMale**
The complete explanation is in DRIVERS-1722. In short, this change is related to the fact that servers <4.9 do not activate fail points configured with `appName`.

DRIVERS-1722

### CODE REVIEW

**Review by ShaneHarvey — APPROVED** [review]
LGTM but I think we should proceed as follows:
- Commit this change as something like "DRIVERS-1722 Skip isMaster-network/command-error tests on 4.4" to more clearly express the motivation.
- Open a new drivers ticket to revert this change once SERVER-49336 is backported and released to 4.4.

This way we won't forget to reenable these tests.

**Comment by stIncMale** [comment]
Thanks, @ShaneHarvey, I filed DRIVERS-1725 (Enable isMaster-network-error, isMaster-command-error tests on servers before 4.9) and renamed DRIVERS-1722 to "Disable isMaster-network-error, isMaster-command-error tests on servers before 4.9".

**Comment by stIncMale** [comment]
How do we usually go about such changes: do we wait for the second reviewer (Neal was added automatically as a code owner) to also approve or leave comments?

**Comment by ShaneHarvey** [comment]
Thanks! We usually wait for the reviewer to either approve or recuse themselves but in this case the change is so small I think you're good to merge whenever.

**Pull request merged**
